src/main/java/com/github/tomakehurst/wiremock/logging/JavaSystemNotifier.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2024 Thomas Akehurst
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.tomakehurst.wiremock.logging;
+
+import static java.lang.System.Logger.Level.ERROR;
+import static java.lang.System.Logger.Level.INFO;
+
+import com.github.tomakehurst.wiremock.common.Notifier;
+
+public class JavaSystemNotifier implements Notifier {
+
+  private final System.Logger log = System.getLogger("WireMock");
+
+  private final boolean verbose;
+
+  public JavaSystemNotifier(boolean verbose) {
+    this.verbose = verbose;
+  }
+
+  @Override
+  public void info(String message) {
+    if (verbose) {
```

Comment:
Why not:

```java
if (logger.isLoggable(Level.INFO)) {
    // ...	
}
```

Let the user configure the logging system however they want it.

Reply:
If there was a lot of processing to happen inside the `if` block it'd be worth it, but since we just log the parameter value we're given there's no performance gain to be had really.

Having said that `if (verbose)` is basically redundant since that parameter is only used in standalone mode which defaults to the console notifier, so I'll remove that.